packages/hop-node/src/cli/stake.ts

```diff
@@ -80,7 +152,33 @@ program
       const chain = source.chain
       const token = source.token
       const amount = Number(source.args[0] || source.amount)
-      await staker(chain, token, amount, StakerAction.Stake)
+      const skipSendToL2 = source.skipSendToL2
+
+      if (!amount) {
+        throw new Error('amount is required. E.g. 100')
+      }
+      if (!chain) {
+        throw new Error('chain is required')
+      }
+      const bridge: L2Bridge | L1Bridge = await getBridge(token, chain)
+      const parsedAmount: BigNumber = bridge.parseUnits(amount)
```

Comment:
do decimals need to be passed here for `parseUnits`?

Reply:
It is my understanding that the `bridge` [does this already](https://github.com/hop-protocol/hop/blob/develop/packages/hop-node/src/watchers/classes/Bridge.ts#L506-L512) with the `token` associated with the bridge itself.

Reply:
you're right. Disregard comment